### PR TITLE
dev/financial#163 Removal of contributionTypeID

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
PR illustrates the remaining places where I propose to remove contributionTypeID. I will include this proposal in the dev-digest so this visualises is

Before
----------------------------------------
Hard to see what would be affected

After
----------------------------------------
PR makes the places clear

Technical Details
----------------------------------------
Depending on feedback this PR might be updated to actually remove the commented lines

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Event/Form/Registration/Confirm.php [hunk 1]
```diff
@@ -563,11 +563,12 @@ public function postProcess() {
           //passing contribution id is already registered.
           $contribution = $this->processContribution($this, $value, $result, $contactID, $pending, $this->_paymentProcessor);
           $value['contributionID'] = $contribution->id;
+          // https://lab.civicrm.org/dev/financial/-/issues/163 proposes removing contributionTypeID,
+          // not used in core.
           $value['contributionTypeID'] = $contribution->financial_type_id;
           $value['receive_date'] = $contribution->receive_date;
           $value['trxn_id'] = $contribution->trxn_id;
           $value['contributionID'] = $contribution->id;
-          $value['contributionTypeID'] = $contribution->financial_type_id;
```
Comment (eileenmcnaughton) [inline CRM/Event/Form/Registration/Confirm.php:570]: yeah we only need one of this & line 568

**Comment by eileenmcnaughton** [comment]
I'm not going to worry about the test fails unless this seems hard to get agreement on - if agreed I'll change to remove, otherwise we should get this passing & committed for next round of this

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by mattwire** [comment]
@eileenmcnaughton Yes definitely remove

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by eileenmcnaughton** [comment]
This is now updated to remove rather than comment the relevant places

**Comment by eileenmcnaughton** [comment]
@colemanw this should be good to merge now

**Comment by eileenmcnaughton** [comment]
First we take contributionTypeID then we take contributionMode

**Pull request merged**